yuptoo/modifiers/transform_tags.py

```diff
@@ -7,6 +7,10 @@ def run(self, host: dict, transformed_obj: dict, **kwargs):
         if tags:
             tags_modified = False
             for tag in tags:
+
+                if len(str(tag['value'])) > 255:
+                    tag['value'] = "Truncated.Longer than maximum length 255."
```

Comment:
```suggestion
                    tag['value'] = "Original value exceeds 250 characters."
```
To have consistency with [foreman_rh_cloud](https://github.com/theforeman/foreman_rh_cloud/blob/master/lib/foreman_inventory_upload/generators/tags.rb#L63), it would be good to have same message here.

Reply:
Addressed this comment in #53.